tests/e2e/specs/modules/tagmanager/setup.test.js

```diff
@@ -169,4 +177,40 @@ describe( 'Tag Manager module setup', () => {
 		await expect( page ).toMatchElement( '.googlesitekit-setup-module--tag-manager .mdc-button', { text: /create an account/i } );
 		await expect( page ).toMatchElement( '.googlesitekit-setup-module--tag-manager .googlesitekit-cta-link', { text: /re-fetch my account/i } );
 	} );
+
+	describe.each(
+		Object.keys( allowedAmpModes )
+	)( 'AMP mode %s', ( mode ) => {
+		beforeEach( async () => {
+			await activatePlugin( 'amp' );
+			await activatePlugin( 'e2e-tests-module-setup-tagmanager-api-mock' );
+		} );
+		afterEach( async () => {
+			await deactivatePlugin( 'amp' );
+		} );
+		it( 'renders the correct drop downs in the setup screen', async () => {
+			await setAMPMode( mode );
+			await proceedToTagManagerSetup();
+
+			// Ensure that the correct dropdowns are displayed.
+			if ( 'standard' !== mode ) {
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--web' );
+			}
+			await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp' );
```

Comment:
Let's also check that:
* The dropdowns are _not_ shown that shouldn't be
    - AMP container select should always be shown when AMP is active
    - The web container select should not be shown for primary AMP
* The labels are correct for each case as there is some logic there.
    - For primary AMP, the label should be "Container" (same as no AMP)
    - For secondary AMP, the labels should be "AMP Container" and "Web Container"

Reply:
Updated